<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RipEmuratorClass</class>
 <widget class="QMainWindow" name="RipEmuratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>562</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>562</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="BtnStart">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BtnStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="chkPoisonReverse">
         <property name="text">
          <string>PoisonReverse</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkHorizonSplit">
         <property name="text">
          <string>HorizonSplit</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>RipInfo</string>
        </attribute>
        <widget class="QTableView" name="TblRipInfo">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>511</width>
           <height>251</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView {
	selection-background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.739, stop:0 rgba(65, 165, 231, 255), stop:1 rgba(255, 255, 255, 255));
    padding:2px;
}
</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>10</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>10</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>History</string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>271</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="chkRecord">
              <property name="text">
               <string>Record</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>517</width>
               <height>236</height>
              </rect>
             </property>
             <widget class="QTextEdit" name="TxtHistory">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>521</width>
                <height>271</height>
               </rect>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>TxtHistory</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
